File: engine/base/src/main/java/client/database/data/CharacterGuildFamilyData.java
package client.database.data;

public class CharacterGuildFamilyData {
   private int world;

   private int guildId;

   private int guildRank;

   private int familyId;

   public CharacterGuildFamilyData(int world, int guildId, int guildRank, int familyId) {
      this.world = world;
      this.guildId = guildId;
      this.guildRank = guildRank;
      this.familyId = familyId;
   }

   public int getWorld() {
      return world;
   }

   public int getGuildId() {
      return guildId;
   }

   public int getGuildRank() {
      return guildRank;
   }

   public int getFamilyId() {
      return familyId;
   }
}

File: engine/engine-model/src/main/scala/client/database/data/CharacterGuildFamilyData.scala
package client.database.data

class CharacterGuildFamilyData(private var _world: Int, private var _guildId: Int, private var _guildRank: Int,
                               private var _familyId: Int) {
  def world: Int = _world

  def guildId: Int = _guildId

  def guildRank: Int = _guildRank

  def familyId: Int = _familyId
}
